static/style.css

```diff
@@ -1372,4 +1457,13 @@ td, th {
 		padding: 7px 0px;
 		margin-right: -5px;
 	}
+
+	.popup {
+		width: auto;
+		bottom: 10vh;
+	}
+	
```

Comment:
It doesn't show here, but `git diff` reports there are unnecessary spaces in what is supposed to be a blank line here.

Reply:
odd. weird that vscode didn't catch it